### ISSUE TITLE
Make invisible characters visible in diff when difference is only in invisible characters

### ISSUE THREAD

**Issue by sebastianbergmann** [sebastianbergmann/phpunit#1804]
## Example

``` php
<?php
class Test extends PHPUnit_Framework_TestCase
{
    public function testOne()
    {
        $this->assertEquals("test\r\n", "test\n");
    }
}
```
## Actual

```
$ phpunit Test
PHPUnit 5.0-g4e1512f by Sebastian Bergmann and contributors.

F

Time: 54 ms, Memory: 4.75Mb

There was 1 failure:

1) Test::testOne
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 'test
 '

/home/sb/Test.php:6

FAILURES!
Tests: 1, Assertions: 1, Failures: 1.
```
## Expected

```
$ phpunit Test
PHPUnit 5.0-g4e1512f by Sebastian Bergmann and contributors.

F

Time: 54 ms, Memory: 4.75Mb

There was 1 failure:

1) Test::testOne
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'test\r\n
+'test\n
 '

/home/sb/Test.php:6

FAILURES!
Tests: 1, Assertions: 1, Failures: 1.
```

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
This would have been every useful to me in the past. :)

**Comment by nned** [comment]
Would be nice!

**Comment by lindt** [comment]
already fixed with #2148
